proxy/coco/src/state.rs

```diff
@@ -577,11 +581,29 @@ impl State {
     where
         P: Into<Option<PeerId>> + Clone + Send + 'static,
     {
-        let proj = self.get_project(urn.clone(), peer_id).await?;
-        let include_path = self.update_include(urn).await?;
+        let proj = self.get_project(urn.clone(), None).await?;
```

Comment:
What's the significance of not passing the peer_id here anymore?

Reply:
We can _always_ get a project without a `PeerId` so I thought that would make sense to get the project metadata that way. But now that I'm thinking about it, it might be significant if the user's project data differs from the peers. If the default branch has changed it might mean that the user tries to checkout the wrong branch. 

So, I think I'll change it back to passing the `peer_id` :+1: